finagle-thrift/src/main/scala/com/twitter/finagle/rich.scala

```diff
@@ -380,7 +396,18 @@ trait ThriftRichClient { self: Client[ThriftClientRequest, Array[Byte]] =>
    * $clientUse
    */
   def newIface[Iface](name: Name, label: String, cls: Class[_]): Iface = {
-    val underlying = newService(name, label)
+    newIface(name, label, cls, protocolFactory, newService(name, label))
+  }
+
+  /**
+   * $clientUse
+   */
+  def newIface[Iface](
+    name: Name,
+    label: String,
+    cls: Class[_],
+    protocolFactory: TProtocolFactory,
+    service: Service[ThriftClientRequest, Array[Byte]]): Iface = {
```

Comment:
style nitpick: move the `): Iface = {` down to the next line, indented to the same as the `def`.